core-bundle/src/Controller/AbstractFragmentController.php

```diff
@@ -114,11 +114,16 @@ protected function addHeadlineToTemplate(Template $template, $headline): void
     /**
      * @param string|array $cssID
      */
-    protected function addCssAttributesToTemplate(Template $template, string $templateName, $cssID, array $classes = null): void
+    protected function addCssAttributesToTemplate(Template $template, string $templateName, $cssID, array $classes = null, string $overrideCssID = null): void
     {
         $data = StringUtil::deserialize($cssID, true);
         $template->class = trim($templateName.' '.($data[1] ?? ''));
-        $template->cssID = !empty($data[0]) ? ' id="'.$data[0].'"' : '';
+
+        if (!empty($overrideCssID)) {
+            $template->cssID = $overrideCssID;
```

Comment:
No surrounding `id=""` markup here?

Reply:
good catch, that certainly is a bug!

Reply:
fixed in 101d7a087c6fff9b7ccdabb184a9f71a7dd2f4cc